### ISSUE TITLE
Docker run failing with `Low disk space. Make sure to have at least 100MB of free disk space.`  error

### ISSUE THREAD

**Issue by shahiddev** [parkpow/deep-license-plate-recognition#79]
**Describe the bug**
When trying to run the current latest [docker image] (https://hub.docker.com/layers/platerecognizer/alpr/latest/images/sha256-e1ff8e02b4616a20a38eeee61ab0bf57a56a42c2d9d96d683649319547ce56bf?context=explore) docker run fails with error 
```
PlateRecognizer v1.27.0
Starting online registration
You have used 2/2 license(s).
License Key: <redacted>
Usage since 2021-12-30: <redacted> calls.
Loading...
Loading detection zones.
Low disk space. Make sure to have at least 100MB of free disk space.
```
**To Reproduce**
Execute Docker run command as per documentation
`docker run --rm -t -p 8080:8080 -v license:/license -e TOKEN=<redacted> -e LICENSE_KEY=<redacted> platerecognizer/alpr`

**Expected behavior**
Container should start rather than terminating immediately

**Image of vehicle**
n/a

**Desktop (please complete the following information):**
 - OS: Windows 10 21H2
 - Version:  19044.1466
 - Hardware: Dell XPS 15 (i7-10875H CPU @ 2.30GHz, 32.0 GB RAM)

**Additional context**
We found the image pushed prior to this latest one was also failing for the same reason (now tagged as `platerecognizer/alpr:22-01-25`. 
Currently the last working version for us running locally or in production is `platerecognizer/alpr:21-12-21`

**Comment by danleyb2** [comment]
@shahiddev do you have enough disk space?

**Comment by shahiddev** [comment]
Yes about 400 GB iirc, we saw the  2022-01-25 container fail in a deployment (yesterday NZ time when it was the latest version) because we weren't pinned to a specific version. Pinning to the Dec version with no other changes allowed the container to start. I can also repro on my local machine reliably

**Comment by danleyb2** [comment]
@shahiddev maybe it's your docker storage limit. What's your OS version?

**Comment by shahiddev** [comment]
I doubt it unless these latest images need more space? I ran the Dec image immediately after seeing the error on the latest version without issue. Will check tomorrow
OS version is in the original post, latest docker desktop version.

**Comment by danleyb2** [comment]
What dies docker use for virtualization on your system? WSL or HyperV? The previous image might continue working because it used lesser space

**Comment by danleyb2** [comment]
@shahiddev

**Comment by shahiddev** [comment]
WSL2 - I'll check the disk space in the morning and report back

**Comment by shahiddev** [comment]
Hi @danleyb2 
My machine has 700GB free not 400GB - so definitely not a local disk issue
Docker desktop is using WSL2 so resource limits are controlled by `.wslconfig` not Docker desktop
My wslconfig only has constraints on memory not disk space

```powershell
[08:50:31] ~ ❯ cat .\.wslconfig
[wsl2]
memory=6GB
```

**Comment by shahiddev** [comment]
I've also tried deleting the `license` volume before starting - same error

**Comment by danleyb2** [comment]
@shahiddev I have a similar docker version and the latest image works okay for me.
Is it only our image that doesn't work? Could you try running another image like ubuntu.

**Comment by shahiddev** [comment]
I've had no issues with other Linux containers, running the Sql server one at the same time. I did spin up a. Ubuntu vm on Azure and latest image was fine so yeah not sure right now. I'll try a few other images and also try yours from my other machines.

**Comment by danleyb2** [comment]
@shahiddev also try switching to a bind mount instead of a named volume
Use ` -v /tmp/user/license2:/license` instead of  `-v license:/license`

**Comment by marcbelmont** [comment]
@shahiddev are you still struggling with this issue?

**Comment by shahiddev** [comment]
Hi @marcbelmont I am on my work machine, I tried the bind mount without success.
I'm going to try on my personal Windows machine to see if that helps understand the problem.

I've tried including -e LOGGING 10 but it doesn't appear to display any additional info, before I try and dig into the code what is the the check doing that is triggering the error.

**Comment by marcbelmont** [comment]
@shahiddev It's looking at how much free space there is. Could you share the output of that command?

Start the container `docker run --rm -ti platerecognizer/alpr bash`
When inside the container run `df -h`. The mount point `/` should have free space. For example,  `overlay         468G  395G   50G  89% /`

**Comment by shahiddev** [comment]
Hey Marc,
As requested:

```
root@0f657c2097ca:/app# df -h
Filesystem      Size  Used Avail Use% Mounted on
overlay         251G  243G     0 100% /
tmpfs            64M     0   64M   0% /dev
tmpfs           2.9G     0  2.9G   0% /sys/fs/cgroup
shm              64M     0   64M   0% /dev/shm
/dev/sdd        251G  243G     0 100% /etc/hosts
tmpfs           2.9G     0  2.9G   0% /proc/acpi
tmpfs           2.9G     0  2.9G   0% /sys/firmware
```

**Comment by shahiddev** [comment]
For comparison when using the 21-12-21 image which does run on this machine

```
root@b97214ed0216:/app# df -h
Filesystem      Size  Used Avail Use% Mounted on
overlay         251G  243G     0 100% /
tmpfs            64M     0   64M   0% /dev
tmpfs           2.9G     0  2.9G   0% /sys/fs/cgroup
shm              64M     0   64M   0% /dev/shm
/dev/sdd        251G  243G     0 100% /license
tmpfs           2.9G     0  2.9G   0% /proc/acpi
tmpfs           2.9G     0  2.9G   0% /sys/firmware
```

**Comment by danleyb2** [comment]
@shahiddev I think your WSL 2 VHD is full. The initial maximum size of 256GB.
Could you try resizing it as explained here:
https://docs.microsoft.com/en-us/windows/wsl/vhd-size
